README.md

```diff
@@ -1,3 +1,36 @@
-This is a nascent project for implementing a rich text layout engine backed
-by the [swash](https://github.com/dfrg/swash) crate and implementing the 
-[piet](https://github.com/linebender/piet) text API.
+<div align="center">
+
+# Parley
+
+**Rich text layout**
+
+[![Latest published parley version.](https://img.shields.io/crates/v/parley.svg)](https://crates.io/crates/parley)
+[![Documentation build status.](https://img.shields.io/docsrs/parley.svg)](https://docs.rs/parley)
+[![Dependency staleness status.](https://deps.rs/repo/github/linebender/parley/status.svg)](https://deps.rs/repo/github/linebender/parley)
+[![Linebender Zulip chat.](https://img.shields.io/badge/Linebender-%23gpu-blue?logo=Zulip)](https://xi.zulipchat.com/#narrow/stream/205635-text)
+[![Apache 2.0 or MIT license.](https://img.shields.io/badge/license-Apache--2.0_OR_MIT-blue.svg)](#license)
```

Comment:
I like the change to alt text with a final `.`.

I've just done some initial digging into the accessibility story around badges, and it looks like it's a bit messy in terms of doing contrast/font size properly. Might be worth revisiting